### PR TITLE
Enforcing the check of env value

### PR DESCRIPTION
**Pull request by rravindran123**
This PR checks the value of the Environmental value in addition to just checking its definition as it is currently implemented.

### CODE REVIEW

File: operators/nephio-controller-manager/main.go [hunk 1]
```diff
@@ -204,14 +204,17 @@ func parseReconcilers(reconcilers string) []string {
...
 	if slices.Contains(reconcilers, "*") {
 		return true
 	}
 	if slices.Contains(reconcilers, reconciler) {
 		return true
 	}
-	if _, found := os.LookupEnv(fmt.Sprintf("ENABLE_%s", strings.ToUpper(reconciler))); found {
-		return true
+	if val, found := os.LookupEnv(fmt.Sprintf("ENABLE_%s", strings.ToUpper(reconciler))); found {
+		if val == "true" {
```
Comment (subh007) [inline operators/nephio-controller-manager/main.go:215]: we could avoid extra check :
```
return val
```
:)

**Comment by johnbelamaric** [comment]
/approve

yes, please

**Review by vjayaramrh — APPROVED** [review]
/lgtm

**Pull request merged**